chronograf.go

```diff
@@ -380,7 +380,7 @@ type Dashboard struct {
 
 // DashboardCell holds visual and query information for a cell
 type DashboardCell struct {
-	ID      string           `json:"-"`
+	ID      string           `json:"i"`
```

Comment:
Yeah actually -- I noticed that it works with strings like `"abcdef-123456-abcdef-123456"` but not with strings like `dashboards/1/cells/abcdef ...`. I'm sure it's not the string that's the problem, but some sort of comparison that's happening in some component somewhere. So turns out my reversion doesn't actually work (which I never pushed).